client/task-list/tasks/payments/methods.js

```diff
@@ -374,6 +376,27 @@ export function getPaymentMethods( {
 				options.woocommerce_razorpay_settings.enabled === 'yes',
 			optionName: 'woocommerce_razorpay_settings',
 		},
+		{
+			key: 'payubiz',
+			title: __( 'PayU for WooCommerce', 'woocommerce-admin' ),
+			content: (
+				<>
+					{ __(
+						'Enable PayU’s exclusive plugin for WooCommerce to start accepting payments in 100+ payment methods available in India including credit cards, debit cards, UPI, & more!',
+						'woocommerce-admin'
+					) }
+				</>
+			),
+			before: <PayUIndiaLogo />,
+			visible: countryCode === 'IN' && ! hasCbdIndustry,
+			plugins: [ 'payu-india' ],
+			container: <PayUIndia />,
+			isConfigured: activePlugins.includes( 'payu-india' ),
+			isEnabled:
+				options.woocommerce_payubiz_settings.enabled &&
+				options.woocommerce_payubiz_settings.enabled === 'yes',
```

Comment:
With the last paypal changes I added `enabledPaymentGateways`, so you could check for `enabledPaymentGateways.includes('payubiz')` instead.

Reply:
good suggestion and thanks for adding that setting. Resolved here: https://github.com/woocommerce/woocommerce-admin/pull/6332/commits/3d81baa0c8db3588d964e907f46fc3a4602e06fa